### ISSUE TITLE
New steam specific installer incompatible with Steam flatpak

### ISSUE THREAD

**Issue by nickavem** [flathub/net.lutris.Lutris#132]
Tl;DR: The new seam specific installers in the "Steam" section do not work, however deprecated community steam installers do.

Steps to reproduce:

1. Have steam and Lutris installed and set up.
2. Have steam either completely closed or open in the background.
3. Attempt to install steam game in Lutris steam section.

Expected behavior:
Steam will launch (if needed) and open the installation menu for the specific game.

Actual Behavior:
Nothing happens except for a trace back in terminal (see below).

Notes:
- This issue could be upstream, but I probably won't test upstream on a VM etc. as I don't completely understand the issue.

Logs:
```sh
Unable to load locale dir, translations won't work.
2020-11-20 15:51:55,127: Couldn't find a terminal emulator.
Gtk-Message: 15:51:55.331: Failed to load module "canberra-gtk-module"
2020-11-20 15:51:55,333: Lutris 0.5.8
2020-11-20 15:51:55,333: Running X.Org Mesa driver 20.1.10 on AMD Radeon RX 5600 XT (NAVI10, DRM 3.39.0, 5.9.8-200.fc33.x86_64, LLVM 10.0.1) (0x731f)
2020-11-20 15:51:55,334: GPU: 1002:731F 1682:5710 (amdgpu drivers)
### Attempt to install steam game
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/lutris/gui/widgets/game_bar.py", line 221, in on_install_clicked
    self.service.install(self.db_game)
  File "/app/lib/python3.8/site-packages/lutris/services/steam.py", line 145, in install
    db_games = get_games(filters={"steamid": appid, "installed": "1"})
  File "/app/lib/python3.8/site-packages/lutris/database/games.py", line 21, in get_games
    return sql.filtered_query(PGA_DB, "games", searches=searches, filters=filters, excludes=excludes, sorts=sorts)
  File "/app/lib/python3.8/site-packages/lutris/database/sql.py", line 162, in filtered_query
    return db_query(db_path, query, tuple(params))
  File "/app/lib/python3.8/site-packages/lutris/database/sql.py", line 110, in db_query
    cursor_execute(cursor, query, params)
  File "/app/lib/python3.8/site-packages/lutris/database/sql.py", line 34, in cursor_execute
    results = cursor.execute(query, params)
sqlite3.OperationalError: no such column: steamid
### close lutris
2020-11-20 15:55:00,339: Shutting down Lutris
```

**Comment by gasinvein** [comment]
Hm, the table `games` in my `~/.var/app/net.lutris.Lutris/data/lutris/pga.db` does have the `steamid` column. Maybe Lutris didn't run migrations for you or something?

**Comment by FakeShemp** [comment]
I'm able to reproduce. Only install and uninstall seems to be affected as playing works as it should.

**Comment by juligreen** [comment]
I'm not able to reproduce this issue anymore, I think it can be closed.

**Comment by legluondunet** [comment]
obsolete issue
